opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingDynamicFeature.java

```diff
@@ -21,16 +21,19 @@
  */
 @Provider
 public class ServerTracingDynamicFeature implements DynamicFeature {
-
     private static final Logger log = Logger.getLogger(ServerTracingDynamicFeature.class.getName());
 
     private Builder builder;
 
+    /**
+     * When using this constructor application has to call {@link GlobalTracer#register} to register
+     * tracer instance. Ideally it should be called in {@link javax.servlet.ServletContextListener}.
+     */
     public ServerTracingDynamicFeature() {
         this(new Builder(GlobalTracer.get()));
     }
 
-    private ServerTracingDynamicFeature(Builder builder) {
+    public ServerTracingDynamicFeature(Builder builder) {
```

Comment:
API brainstorm:

Should be this private and only accessible by `Builder.build()`. I have made it public to be easier to find as the default constructor is public too.

@jpkrohling @objectiser thoughts?